Make FAQ Responsive.

ayushx007 opened the pull request:
![WhatsApp Image 2024-01-10 at 10 02 51 PM](https://github.com/dscvitc/Devshouse/assets/113995024/0c8b7ab1-c25c-400a-8667-d38fbe25434b)

ayushx007 committed:
FAQ Responsiveness Fix

ayushx007 committed:
Reponsiveness fix

ayushx007 committed:
Final responsive commit

ayushx007 committed:
Minor animation fix

vaibhavTekk commented:
can you add a little bit of padding on the sides, too close to the edges when on mobile. otherwise lgtm!

ayushx007 commented:
> can you add a little bit of padding on the sides, too close to the edges when on mobile. otherwise lgtm!

Sure, I will give it a try!

ayushx007 committed:
Add Sponsor Us section, insert custom backgrounds and increase FAQ width on small devices

ayushx007 committed:
Fix build issues

ayushx007 committed:
Sponsor Us Responsiveness Added

ayushx007 committed:
Fix

ayushx007 commented:
> can you add a little bit of padding on the sides, too close to the edges when on mobile. otherwise lgtm!

![image](https://github.com/dscvitc/Devshouse/assets/113995024/054ae151-f2ca-43fe-852e-d3a0be28aa1d)
![WhatsApp Image 2024-01-11 at 10 37 06 PM](https://github.com/dscvitc/Devshouse/assets/113995024/2f062dea-3253-4894-87d2-74829dc3d826)
Added padding to FAQ on mobile screens, added the sponsor us section, added responsiveness to the sponsor us section, added custom bg images for the two sections and fixed some build errors. The merge conflicts is because I removed the extra "our teams" section in my code(it appears twice in the current website).
